### PR TITLE
docs(accordion): decompose Dynamic accordion and fix e2e tests

### LINKED ISSUE #3627: docs(accordion): decompose Dynamic accordion
Decompose `Dynamic accordion` to:
- `Dynamic accordion`
- `Manual toggle`
- `Dynamic body content`

related to #3128

### PR DESCRIPTION
**Pull request by artolshansky**
Decompose to: Dynamic accordion, Dynamic body content, Manual toggle; fix e2e tests

close #3627

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated tests.
 - [x] added/updated demos.

### CODE REVIEW

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

File: demo/src/app/components/+accordion/demos/manual-toggle/manual-toggle.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-accordion-manual-toggle',
+  templateUrl: './manual-toggle.html'
+})
+export class DemoAccordionManualToggleComponent {
+  status: any = {
```
Comment (valorkin) [inline demo/src/app/components/+accordion/demos/manual-toggle/manual-toggle.ts]: object for one fild

**Review by valorkin — CHANGES_REQUESTED** [review]

File: demo/src/app/components/+accordion/demos/manual-toggle/manual-toggle.ts (hunk 1 above)
Comment (artolshansky) [reply demo/src/app/components/+accordion/demos/manual-toggle/manual-toggle.ts]: fixed

**Review by IlyaSurmay — DISMISSED** [review]

File: demo/src/app/components/+accordion/demos/dymanic/dynamic.ts [hunk 2]
```diff
@@ -5,13 +5,6 @@ import { Component } from '@angular/core';
...
 })
 export class DemoAccordionDynamicComponent {
-  items: string[] = ['Item 1', 'Item 2', 'Item 3'];
-
-  status: any = {
-    isFirstOpen: true,
-    isOpen: true
-  };
-
   groups: any[] = [
```
Comment (IlyaSurmay) [inline demo/src/app/components/+accordion/demos/dymanic/dynamic.ts]: remove type annotation

File: demo/src/app/components/+accordion/demos/dynamic-body/dynamic-body.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-accordion-dynamic-body',
+  templateUrl: './dynamic-body.html'
+})
+export class DemoAccordionDynamicBodyComponent {
+  items: string[] = ['Item 1', 'Item 2', 'Item 3'];
```
Comment (IlyaSurmay) [inline demo/src/app/components/+accordion/demos/dynamic-body/dynamic-body.ts]: same
if type annotation isn't provided, typescript will use the type of the first assigned value

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — APPROVED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Review by valorkin — APPROVED** [review]

**Pull request merged**